libraries/AC_AutoTune/AC_AutoTune_Heli.h

```diff
@@ -45,6 +45,13 @@ class AC_AutoTune_Heli : public AC_AutoTune
     //
     // methods to load and save gains
     //
+    // sweep_info contains information about a specific test's sweep results
```

Comment:
Move this structure to the private section with the other structures and the enumeration